clis/teliod/src/interface_configurator.rs

```diff
@@ -0,0 +1,120 @@
+use crate::TeliodError;
+use serde::Deserialize;
+use std::net::IpAddr;
+use std::process::Command;
+use tracing::{error, info};
+
+pub trait InterfaceConfigurator {
```

Comment:
NIT: trait name should tell more about functionality provided, not describe an object, thing From, Into vs Converter, Diplay vs Fromatter etc. So i would recommend to name this "ConfigureInterface" vs some "Xinator"

Reply:
Renamed trait to `ConfigureInterface` as well as implementations.